crates/resolver/src/system_conf/unix.rs

```diff
@@ -52,7 +52,11 @@ fn into_resolver_config(
     parsed_config: resolv_conf::Config,
 ) -> io::Result<(ResolverConfig, ResolverOpts)> {
     let domain = if let Some(domain) = parsed_config.get_system_domain() {
-        Some(Name::from_str(domain.as_str())?)
```

Comment:
A more concise way to express this:
```
Name::from_str(domain.as_str()).ok()
```

Reply:
Way better, thanks!

Reply:
The only issue I see with the `ok` usage here is that it will mask any parse failures for the domain.